server/test/testFaceRestUser.cc

```diff
@@ -363,17 +363,23 @@ void test_deleteUser(void)
 {
 	startFaceRest();
 
-	const UserIdType userId = findUserWith(OPPRVLG_DELETE_USER);
-	string url = StringUtils::sprintf("/user/%" FMT_USER_ID, userId);
+	const UserIdType deleterId = findUserWith(OPPRVLG_DELETE_USER);
+	const UserIdType targetId = 1;
+
+	// When deleterId and targetId are same, this test case will fail.
+	// Because, Login user can't delete myself.
+	// If the test fails, you need to fix Users data of DBTableTest.cc.
```

Comment:
I feel something is wrong for the line.
If the test fails in the future, the wrong fix will have to be done.